pages/docs/user-docs/action-build.md

```diff
@@ -1,303 +1,185 @@
-    -T|--notest     Bootstrap without running tests in %test section
-    -s|--section    Only run a given section within the bootstrap (setup,
-                    post, files, environment, test, labels, none)
-
+$ sudo singularity shell --writable lolcow/
 ```
 
-You can think of the different options as follows:
+## Converting containers from one format to another
+If you already have a container saved locally, you can use it as a target to build a new container. This allows you convert containers from one format to another.  For example if you had a squashfs container called `production.simg` and wanted to convert it to a writeable ext3 container called `development.img` you could:
 
-**create writable development folder**
 ```
-sudo singularity build --writable container/ Singularity
+$ singularity build --writable development.img production.simg
 ```
 
-**create writable development ext3 image**
-```
-sudo singularity build --writable container.img Singularity
-```
+Similarly, to convert it to a writable directory (a sandbox):
 
-**create production immutable image**
 ```
-sudo singularity build container.simg Singularity
+$ singularity build --sandbox development/ production.simg
 ```
 
-For details about the Singularity flow and use cases, see our [quickstart](/quickstart).
-
-## Checks
-Next, we will look at checks. Checks haven't been robustly developed (yet!) but offer
-an easy way for an admin to define a security (or any other kind of check) to be run on demand
-for a Singularity image. They are defined (and run) via different tags.
+If you omit any options you can also convert your sandbox back to a read-only compressed squashfs image suitable for use in a production environment:
 
 ```
-CHECKS OPTIONS:
-    -c|--checks    enable checks
-    -t|--tag       specify a check tag (not default)
-    -l|--low       Specify low threshold (all checks, default) 
-    -m|--med       Perform medium and high checks
-    -h|--high      Perform only checks at level high
-
-See singularity check --help for available tags
+$ singularty build production2 development/
 ```
+You can convert the three supported container formats using any combination.
 
+Use care when converting writable ext3 images or sandbox directories to the default squashfs file format.  If changes were made to the writable container before conversion, there is no record of those changes in the Recipe file rendering your container non-reproducible.  It is a best practice to build your immutable production containers directly from a Recipe file instead.
 
-## Examples
-Let's quickly look at some definition file examples. These come straight from `singularity help build`. First, here
-is a recipe for build that serves only to describe the sections.
+## Building containers from Recipe files
+Of course, Recipe files can be used as the target when building a container.  For detailed information on writing Recipe files, please see [the Container Recipes docs](docs-recipes).
 
```

Comment:
The standard we've been using, in many places, Singularity Hub, docs, is "Singularity recipe" or build recipe.

Reply:
The standard we've been using in the help is definition file.  And when I give talks and teach workshops that's what I've been calling it too.  I understand that you like recipe now but I didn't realize it had become a standard. lol

Reply:
I guess you haven't used Singularity Hub. I't's also, erm, published!

![image](https://user-images.githubusercontent.com/814322/31261775-2d07a49c-aa0b-11e7-84e1-5e782ce90996.png)
http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0177459